cobigen-cli/pom.xml

```diff
@@ -73,6 +73,26 @@
         <artifactId>commons-text</artifactId>
         <version>1.6</version>
     </dependency>
+    <dependency>
+  <groupId>com.google.googlejavaformat</groupId>
+  <artifactId>google-java-format</artifactId>
+  <version>1.7</version>
+</dependency>
+<dependency>
+  <groupId>com.google.googlejavaformat</groupId>
+  <artifactId>google-java-format</artifactId>
+  <version>1.7</version>
+</dependency>
+<dependency>
+    <groupId>com.google.guava</groupId>
+    <artifactId>guava</artifactId>
+    <version>23.6-jre</version>
+</dependency> 
+<dependency>
+    <groupId>org.apache.httpcomponents</groupId>
+    <artifactId>httpcore</artifactId>
+    <version>4.4.8</version>
+</dependency>
```

Comment:
Please format correctly the pom file. The indentation should be the same for every dependency.